chart/kubeapps/templates/kubeapps-jobs-upgrade.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1
```

Comment:
I think we should add a comment in this file with:

* An explanation on why this is needed, i.e `upgrade from a previous version that did not support existing secret`
* What it does, create secret, sync repos.
* Mention that this is temporary and add a link to a task to remove it